main.js

```diff
@@ -20,21 +18,23 @@ $('#post-button').click(() => {
   };
   posts.push(newPost);
   renderPosts();
+  //empty the input boxes to prepare for the next post
   $('#message').val("");
   $('#name').val("");
 })
 
-//builds the top line for each post with remove post & comments show/hide links
+//builds the top line for each post with remove, comments show/hide, edit, & full page view links
 var createTopLine = (post => {
```

Comment:
There isn't a need for '(' on the left of your post parameter here. Because there's only one parameter, you can omit the parentheses all together, or you can still use them, just make sure to close the other side.